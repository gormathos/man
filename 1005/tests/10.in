8 2 -10 5 9
-6 2 -3 7 4
9 3 6 -5 2
