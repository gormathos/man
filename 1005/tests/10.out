3
8 2 -10
-6 2 -3
9 3 6
546
